### PR TITLE
fix: remove remains of playground v1

### PR DESCRIPTION
**Pull request by axel7083**

### CODE REVIEW

File: packages/shared/Messages.ts [hunk 1]
```diff
@@ -17,15 +17,6 @@
  ***********************************************************************/
 
 export enum Messages {
-  /**
```
Comment (jeffmaury) [inline packages/shared/Messages.ts:20]: Seems unrelated ?

Comment (axel7083) [reply packages/shared/Messages.ts:20]: There are messages which where used in the Playground v1, there are now useless, as shown by the typecheck not raising any error (my editor was telling me there were unused also)

**Review by jeffmaury — APPROVED** [review]
LGTM
